content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/least_packages.rst

```diff
@@ -0,0 +1,77 @@
+======================
+Least packages removal
+======================
+
+The *Least Packages* removal strategy fulfills an order by opening the fewest number of packages,
+which is ideal for maintaining organized stock without needing to open multiple boxes.
+
+To understand how the removal strategy works, consider the following example, featuring a warehouse
+that stores Packages of Flour in bulk.
+
+Purchased products are packaged in: `10 kg`, `30 kg`, and `50 kg`.
```

Comment:
Why do we need to say the volumes that flour is initially purchased in? All that matters is the volume of the packages they're stored in.